tests/framework/matcher/phase_test.go

```diff
@@ -59,4 +93,33 @@ var _ = Describe("Matcher", func() {
 		table.Entry("cope with a non-stringable object as expected phase", nil, []*v1.Pod{runningPod}, false),
 		table.Entry("with expected phase not match the pod phase", "Succeeded", []*v1.Pod{runningPod}, false),
 	)
+
```

Comment:
Maybe add a case when only `name` is set

Reply:
Sure. Thanks